packages/react-ui/src/components/molecules/UploadField.js

```diff
@@ -167,23 +182,15 @@ function UploadField(props) {
 }
 
 UploadField.defaultProps = {
-  placeholder: 'Drag and drop your file or click to browse',
-  dragPlaceholder: 'Drop your file here...',
   buttonText: 'Browse',
   accept: ['application/JSON'],
-  multiple: false,
-  error: false,
   files: [],
   onChange: (files) => files
 };
 
 UploadField.propTypes = {
-  placeholder: PropTypes.string,
-  dragPlaceholder: PropTypes.string,
   buttonText: PropTypes.string,
   accept: PropTypes.array,
-  multiple: PropTypes.bool,
```

Comment:
`multiple` is still used in the component, so shouldn't be removed here.
Also, looks like it's missing `size`.

Reply:
I recovered back the PropTypes you mentioned 👍